### PR TITLE
Document NICs built into many Acer laptops and some Asus

### PR DESCRIPTION
**Pull request by cschuber**
motherboards.

### CODE REVIEW

**Comment by cschuber** [comment]
These two pull requests document popular NICs in Acer laptops and Asus motherboards.

**Comment by cschuber** [comment]
Missed a line.

File: fcp-0101.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ fxp    | Popular device long recommended by the project.
 dc     | Popular device for CardBus card.
 hme    | Built in interface on many supported sparc64 platforms.
 le     | Emulated by QEMU, alternatives don't yet work for mips64.
+nfe    | nfe is a popular 1GB device in many Ausus motherboards
```
Comment (brooksdavis) [inline fcp-0101.md:55]: Remove this line.  No exception is required as nfe shouldn't have been on the list.

**Review by brooksdavis — CHANGES_REQUESTED** [review]

File: fcp-0101.md (hunk 1 above)
Comment (cschuber) [reply fcp-0101.md:55]: Thanks. I'll review/update my patch and push it at noon my time (PDT) today.

**Comment by cschuber** [comment]
Closing this for now, will discuss in another thread.

**Pull request closed without merging**